lms/static/scripts/postmessage_json_rpc/test/server-test.js

```diff
@@ -268,4 +324,11 @@ describe('Server', () => {
       method,
     };
   }
+
+  function validNotificationRequest(method = 'registeredMethodName') {
```

Comment:
I suggest to put this nearer the top of the file, above where it is used, as that's just more conventional in our code.

Reply:
But but but excuses I was just following what was already there :P 

I've moved the whole set o' functions up.